ui/app/templates/vault/cluster/auth.hbs

```diff
@@ -17,6 +17,36 @@
     </EmptyState>
   </div>
 {{else}}
+  {{#if this.customMessages.bannerMessages.length}}
+    {{#each this.customMessages.bannerMessages as |bannerMessage|}}
+      <Hds::Alert @type="inline" @color="warning" data-test-alert={{bannerMessage.id}} as |A|>
```

Comment:
Do we want spacing between multiple banners? If so you can use a wrapper div styling similar to `.cluster-banners-wrapper`

Reply:
I _think_ HCP has them stacked on top of each other without spacing. I'll confirm with Yongqi!